.github/workflows/e2e.yaml

```diff
@@ -32,7 +28,13 @@ jobs:
       - name: Setup Helm
         uses: fluxcd/pkg/actions/helm@main
       - name: Run tests
-        run: make test
+        run: |
+          docker run --network host \
+                     --volume $(pwd):$(pwd) \
+                     --volume ~/go/pkg/mod:/root/go/pkg/mod \
+                     --env KUBEBUILDER_ASSETS=$KUBEBUILDER_ASSETS \
+                     --workdir $(pwd) golang:1.15-alpine /bin/sh -c \
+                     "apk add --no-cache build-base git ; make test"
```

Comment:
I actually had to add git cause it was complaining that the `git` binary could not be located.